File: PRAK604-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>
#include <string.h>

int main(){
  char kalimat1[50], kalimat2[50];
  
  printf(" ");
  gets(kalimat1);
  printf(" ");
  gets(kalimat2);

  int a = strlen(kalimat1);
  int b = strlen(kalimat2);

  if(a == b){
    int bintang = 0, pagar = 0;
    for(int i = 0; i < a; i++){
      if (kalimat1[i]== ' '&&kalimat2[i]== ' '){
        printf(" ");
      }
      else if(kalimat1[i] == kalimat2[i]){
        printf("*");
        bintang++;
      }
      else{
        printf("#");
        pagar++;
      }
    }
    printf("\n");
   
      printf("* = %d\n# = %d\n", bintang, pagar);
      if (bintang >= pagar){
        printf("Pesan Asli");
      }
      else{
        printf("Pesan Palsu");
      }
  }
  else{
    printf("Panjang kalimat berbeda, pesan palsu");
  }
  return 0;
}

File: PRAK604-2310817310008-MUHAMMADRIZKIRAMADHAN.py
kalimat1 = input(" ")
kalimat2 = input(" ")

a = len(kalimat1)
b = len(kalimat2)

if(a == b):
    bintang = 0; pagar = 0
    for i in range(0, a):
        if kalimat1[i] == ' ' and kalimat2[i] == ' ':
             print(" ", end=" ")
        elif kalimat1[i] == kalimat2[i]:
             print("*", end="")
             bintang += 1
        else:
             print("#", end="")
             pagar += 1

    print("\n* = {}\n# = {}".format(bintang, pagar))
    if bintang >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")
else:
        print("Panjang kalimat berbeda, pesan palsu")